docs/requestAwareMiddleware.md

```diff
@@ -0,0 +1,45 @@
+# The "Request Aware" middleware.
+
+Some middleware (such as the "static" middleware included in this package) do not keep any kind of state variables that
```

Comment:
Should be clearer - ALL middleware must not keep any state variables which can be exposed between requests